data/data/aws/vpc/sg-master.tf

```diff
@@ -118,24 +118,24 @@ resource "aws_security_group_rule" "master_ingress_flannel_from_worker" {
   to_port   = 4789
 }
 
-resource "aws_security_group_rule" "master_ingress_node_exporter" {
+resource "aws_security_group_rule" "master_ingress_internal_from_masters" {
   type              = "ingress"
   security_group_id = "${aws_security_group.master.id}"
 
   protocol  = "tcp"
-  from_port = 9100
-  to_port   = 9100
+  from_port = 9000
+  to_port   = 9990
   self      = true
 }
 
-resource "aws_security_group_rule" "master_ingress_node_exporter_from_worker" {
+resource "aws_security_group_rule" "master_ingress_internal_from_cluster" {
```

Comment:
Can we leave this `from_worker` (vs. `from_cluster`) to match the name used in `security_group_id`?  I'm not against renaming "worker" (e.g. to "compute", #264), but if we do that I'd rather do it more comprehensively than this one name here ;).

Reply:
no objection, I was trying to convey the rule would be for all compute.  is there a more generic selector I should have which is "everything"?

Reply:
> ... is there a more generic selector I should have which is "everything"?

Well, this rule is for everything but the masters.  Our word for that now is "worker"; there are probably better words, and I'm fine with moving to them, but would prefer the worker -> XXX name change happened in larger chunks (e.g. at least throughout AWS Terraform, vs. a partial rename within AWS Terraform).

Reply:
Ok

Reply:
Tweaked naming, opinion?

Reply:
Fine with me.